### PR TITLE
rewriting customer support ad

### PR DESCRIPTION
**Pull request by moyni**
@bamesjerry thoughts?

### CODE REVIEW

File: app/pages/about/jobs/positions/customer-support.js [hunk 1]
```diff
@@ -12,27 +12,66 @@ export default class CustomerSupport extends React.Component {
...
+          satisfied. You will join our friendly, tight-knit team and work together
+          to find creative solutions to even the toughest problems. Ultimately,
+          we want every customer to hang up happy.
+        </p>
+        <p className='para'>
+          A typical day will include:
+        </p>
+        <ul className='list'>
+          <li>answering calls and emails from our customers</li>
+          <li>using our sophisticated in-house fraud detection systems to protect
```
Comment (bamesjerry) [inline app/pages/about/jobs/positions/customer-support.js]: Change *fraud detection* to *risk analysis*.

File: app/pages/about/jobs/positions/customer-support.js [hunk 2]
```diff
@@ -12,27 +12,66 @@ export default class CustomerSupport extends React.Component {
...
+            our users</li>
+          <li>verifying new user accounts</li>
+          <li>talking to new users to help them get the most from our product</li>
+          <li>working with our product team to prioritise feature requests</li>
+        </ul>
+        <p className='para'>
+          You’ll also be involved in improving our internal support tools and
+          processes. We believe that technology should make us more effective,
+          so a lot of our tools are build in-house and regularly updated. As
```
Comment (bamesjerry) [inline app/pages/about/jobs/positions/customer-support.js]: Typo - tools are *built* in-house...

File: app/pages/about/jobs/positions/customer-support.js [hunk 3]
```diff
@@ -12,27 +12,66 @@ export default class CustomerSupport extends React.Component {
...
+          our product is constantly evolving and new features are built, our
+          internal admin dashboard needs regular revisiting. As we learn more
+          about our growing customer base we can optimise our fraud detection
```
Comment (bamesjerry) [inline app/pages/about/jobs/positions/customer-support.js]: _risk analysis_ again.

**Comment by bamesjerry** [comment]
Noice! Just a few minor comments. @jacobpgn ?

**Comment by moyni** [comment]
Thanks @bamesjerry - those are in, plus a couple of other minor rephrasings.
@jacobpgn any comments?

**Comment by jacobpgn** [comment]
Awesome :fire: :+1:

**Pull request merged**